.github/workflows/govulncheck.yaml

```diff
@@ -0,0 +1,37 @@
+on:
+  workflow_call:
+
+name: Govulncheck check
+
+permissions:
+  contents: read
+
+jobs:
+  govulncheck:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install golang
+        run: |
+          ./tests/install_go.sh -f -p
+          echo "/usr/local/go/bin" >> "${GITHUB_PATH}"
+
+      - name: Install govulncheck
+        run: |
+          go install golang.org/x/vuln/cmd/govulncheck@latest
+
+      - name: Build all runtime binaries
+        run: |
+          cd src/runtime
+          make all
+        env:
+          SKIP_GO_VERSION_CHECK: "1"
+
+      - name: Run govulncheck on all runtime binaries
```

Comment:
Let's further refine this step: the `govulncheck.sh` script should take a path to a single binary, and from the workflow we use `matrix:` to run the script on multiple binaries. Will make the output much easier to read and the script simpler.

Reply:
That's a good idea. But if we modify the script to accept a single binary, we lose the ability to filter out all false positives or resolved CVEs in one place. Different binaries could have different false positives. Therefore, we would need to implement that function in the YAML file or create another script, which could be unnecessarily complex.

Reply:
Got it. I initially considered an overall summary, but it's not necessary. I've refactored the script to accept a single binary and used a matrix to run checks across binaries.